digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -489,6 +504,10 @@ ObjectNode fetchLogEvents(final EcsClient client,
         final Optional<String> previousToken = !previousExecutorStatus.has("next_token") ?
                 Optional.absent() : Optional.of(previousExecutorStatus.get("next_token").asText());
         final GetLogEventsResult result = client.getLog(toLogGroupName(previousStatus), toLogStreamName(previousStatus), previousToken);
+        if (result == null) {
+            return previousExecutorStatus.deepCopy();
```

Comment:
`fetchLogEvents()` changed to not throw exception when LogStream is not found.

If container tasks exit non-zero, LogStream may not be created. Then, fall through to https://github.com/treasure-data/digdag/blob/v0_11/digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java#L394 without showing application stdout/stderr (no entry in Cloudwatch logs).